packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -310,6 +313,23 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         }
     }
 
+    public async tombstone() {
```

Comment:
Should a tombstoned data store be marked as dispose as well? For all intent and purposes these are the same - The data store is now allowed to be used in any way.

Reply:
I'm going to think about this some more.

Reply:
If tombstone acts exactly like dispose, that would mean we would need to also add logic to the setConnectionState in the container runtime to detect and skip loading contexts that are disposed or tombstoned. In my opinion, it muddies the difference between dispose and tombstone. I think these are two different behaviors, as we want to error on dispose, but the only time we would error on tombstone is when the datastore is externally loaded by a user, not the internal runtime.

Reply:
Good point. I think we need to think about it in terms of what would have happened if this data store was deleted since tombstone is just a mock delete. Anything that would fail if the data store was deleted should fail and others should not. setConnectionState is a good example of something that would not fail because the call would never reach the data store context.